src/MarkdownTextInput.web.tsx

```diff
@@ -236,6 +238,40 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
       [parser, parseText, processedMarkdownStyle],
     );
 
+    const handleFormatSelection = useCallback(
+      (target: MarkdownTextInputElement, parsedText: string, cursorPosition: number, formatCommand: string): ParseTextResult => {
+        if (!contentSelection.current) {
+          return {
+            text: '',
+            cursorPosition: 0,
+          };
+        }
+
+        let formatType: FormatType;
+        switch (formatCommand) {
+          case 'formatBold':
+            formatType = 'bold';
+            break;
+          case 'formatItalic':
+            formatType = 'italic';
+            break;
+          default:
+            formatType = 'underline';
+            break;
+        }
```

Comment:
Why `underline` is set as default?

Reply:
I actually asked about this [here](https://github.com/Expensify/react-native-live-markdown/pull/556#discussion_r1873492458). Since we only handle the bold, italic, and underline command, I default it to underline. Should I use the "throw" approach?